From a5e5b0638c744677a09800327e48ee5923249666 Mon Sep 17 00:00:00 2001
From: Termux Build <termux@local>
Date: Fri, 24 Jul 2026 19:21:36 +0800
Subject: [PATCH] 0008: exec anti-anti-frida

---
 src/embed-agent.py | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/src/embed-agent.py b/src/embed-agent.py
index cfb4a594..84c64ed6 100644
--- a/src/embed-agent.py
+++ b/src/embed-agent.py
@@ -3,6 +3,7 @@ import shutil
 import subprocess
 import sys
 import struct
+import os
 
 
 def main(argv):
@@ -78,6 +79,14 @@ def main(argv):
                 shutil.copy(agent, embedded_agent)
             else:
                 embedded_agent.write_bytes(b"")
+
+            custom_script = str(output_dir) + "/../../../../frida/subprojects/frida-core/src/anti-anti-frida.py"
+            return_code = os.system("python3 " + custom_script + " " + str(priv_dir / f"frida-agent-{flavor}.so"))
+            if return_code != 0:
+                print("anti-anti-frida error. Code:", return_code)
+            else:
+                print("anti-anti-frida finished")
+
             embedded_assets += [embedded_agent]
     elif host_os in {"freebsd", "qnx"}:
         embedded_agent = priv_dir / "frida-agent.so"
-- 
2.55.0

