From 5f92da1e62ce56ad43f8491f154801c1bb729c31 Mon Sep 17 00:00:00 2001
From: Termux Build <termux@local>
Date: Fri, 24 Jul 2026 19:10:54 +0800
Subject: [PATCH] 0014: temp-prefix

---
 src/system.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/system.vala b/src/system.vala
index 998ef8cc..e14bd59a 100644
--- a/src/system.vala
+++ b/src/system.vala
@@ -238,7 +238,7 @@ namespace Frida {
 		}
 
 		public static string make_name () {
-			var builder = new StringBuilder ("frida-");
+			var builder = new StringBuilder ("gcore-");
 			for (var i = 0; i != 16; i++)
 				builder.append_printf ("%02x", Random.int_range (0, 256));
 			return builder.str;
-- 
2.55.0

